src/Layer/FeatureGeometryLayer.js

```diff
@@ -22,22 +22,37 @@ class FeatureGeometryLayer extends GeometryLayer {
      * @param {string} id - The id of the layer, that should be unique. It is
      * not mandatory, but an error will be emitted if this layer is added a
      * {@link View} that already has a layer going by that id.
-     * @param {Object} [config] - Optional configuration, all elements in it
+     * @param {Object} [options] - Optional configuration, all elements in it
      * will be merged as is in the layer.
      * @param {function} [options.batchId] - optional function to create batchId attribute.
      * It is passed the feature property and the feature index.
      * As the batchId is using an unsigned int structure on 32 bits, the batchId could be between 0 and 4,294,967,295.
-     * @param {THREE.Object3D} [config.object3d=new THREE.Group()] root object3d layer.
+     * @param {THREE.Object3D} [options.object3d=new THREE.Group()] root object3d layer.
+     * @param {boolean} [options.useProj4=FALSE] this option selects the method to transform data to crs view.
+     * if `useProj4` is `true`, `proj4` is used to transform datas source. Use `proj4` so
+     * the transformation will be more geographically precise.
+     *
+     * if `useProj4` is `false`, a affine transformation is used to transform datas source.
+     * this method is an approximation. The error increases with the extent
+     * dimension of the object or queries
      *
      */
-    constructor(id, config = {}) {
-        config.update = FeatureProcessing.update;
-        config.convert = Feature2Mesh.convert({
-            batchId: config.batchId,
+    constructor(id, options = {}) {
+        options.update = FeatureProcessing.update;
+        options.convert = Feature2Mesh.convert({
+            batchId: options.batchId,
         },
         );
-        super(id, config.object3d || new Group(), config);
+        super(id, options.object3d || new Group(), options);
         this.isFeatureGeometryLayer = true;
+        this.useProj4 = options.useProj4 ?? false;
```

Comment:
Shouldn't it be true by default ? So it is by default more precise geographically but you can use an option to increase performances ?